### PR TITLE
Improve test coverage

### PR DESCRIPTION
**Pull request by Zorii4**
- [ ] **Breaking change?** (if so, please describe the impact and migration path for existing applications:)

**What changes did you make?** (Give an overview)
Improved test coverage some files from Topic component
**Is there anything you'd like reviewers to focus on?**

**How Has This Been Tested?** (put an "X" next to an item)

- [ ] No need to
- [x] Manually(please, describe, when necessary)
- [x] Unit checks
- [ ] Integration checks
- [ ] Covered by existing automation

**Checklist** (put an "X" next to an item, otherwise PR will fail)
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation (e.g. **ENVIRONMENT VARIABLES**)
- [x] My changes generate no new warnings(e.g. Sonar is happy)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged

Check out [Contributing](https://github.com/provectus/kafka-ui/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/provectus/kafka-ui/blob/master/CODE-OF-CONDUCT.md)

### CODE REVIEW

File: kafka-ui-react-app/src/components/Topics/Topic/Details/Messages/__test__/utils.spec.ts [hunk 1]
```diff
@@ -30,4 +39,82 @@ describe('utils', () => {
...
+      searchParams.set('seekType', SeekType.TIMESTAMP);
+      searchParams.delete('seekTo');
+      expect(getTimestampFromSeekToParam(searchParams)).toEqual(new Date(0));
+    });
+  });
+
+  describe('getSelectedPartitionsFromSeekToParam', () => {
+    const part: Partition[] = [{ partition: 42, offsetMin: 0, offsetMax: 100 }];
+
+    it('return when serachParams have seekTo and PartititionId includes selected partition', () => {
```
Comment (workshur) [inline kafka-ui-react-app/src/components/Topics/Topic/Details/Messages/__test__/utils.spec.ts:101]:
```suggestion
    it('returns parsed partition from params when partition list includes selected partition', () => {
```

File: kafka-ui-react-app/src/components/Topics/Topic/Details/Messages/__test__/utils.spec.ts [hunk 2]
```diff
@@ -30,4 +39,82 @@ describe('utils', () => {
...
+      searchParams.set('seekTo', '42::0');
+      expect(getSelectedPartitionsFromSeekToParam(searchParams, part)).toEqual([
+        { label: '42', value: 42 },
+      ]);
+    });
+    it('return when serachParams have seekTo and PartititionId NOT includes selected partition', () => {
```
Comment (workshur) [inline kafka-ui-react-app/src/components/Topics/Topic/Details/Messages/__test__/utils.spec.ts:107]: update description

File: kafka-ui-react-app/src/components/Topics/Topic/Details/Messages/__test__/utils.spec.ts [hunk 3]
```diff
@@ -30,4 +39,82 @@ describe('utils', () => {
...
+      searchParams.set('seekTo', '24::0');
+      expect(getSelectedPartitionsFromSeekToParam(searchParams, part)).toEqual(
+        []
+      );
+    });
+    it('return when searchParams not have seekTo', () => {
```
Comment (workshur) [inline kafka-ui-react-app/src/components/Topics/Topic/Details/Messages/__test__/utils.spec.ts:113]: returns what?

File: kafka-ui-react-app/src/components/Topics/Topic/Details/Settings/ConfigListItem.tsx [hunk 4]
```diff
@@ -3,19 +3,23 @@ import React from 'react';
...
-interface ListItemProps {
+export interface ListItemProps {
   config: TopicConfig;
 }
 
 const ConfigListItem: React.FC<ListItemProps> = ({
   config: { name, value, defaultValue },
 }) => {
-  const hasCustomValue = value !== defaultValue;
+  const hasCustomValue = !!defaultValue && value !== defaultValue;
```
Comment (workshur) [inline kafka-ui-react-app/src/components/Topics/Topic/Details/Settings/ConfigListItem.tsx:13]: 👍

File: kafka-ui-react-app/src/components/Topics/Topic/Details/Settings/Settings.styled.ts [hunk 5]
```diff
@@ -6,5 +6,5 @@ export const ConfigList = styled.tr`
   }
 `;
 export const ConfigItemCell = styled.td<{ $hasCustomValue: boolean }>`
-  font-weight: ${(props) => (props.$hasCustomValue ? 500 : 400)};
+  font-weight: ${(props) => (props.$hasCustomValue ? 500 : 400)} !important;
```
Comment (workshur) [inline kafka-ui-react-app/src/components/Topics/Topic/Details/Settings/Settings.styled.ts:9]: I'm ok with it

File: kafka-ui-react-app/src/components/Topics/Topic/Details/Messages/__test__/utils.spec.ts [hunk 6]
```diff
@@ -30,4 +39,82 @@ describe('utils', () => {
...
+    });
+  });
+
+  describe('getTimestampFromSeekToParam', () => {
+    beforeEach(() => {
+      paramsString = `seekTo=0::1627333200000,1::1627333200000`;
+      searchParams = new URLSearchParams(paramsString);
+    });
+
+    it('returns when seekType is equal BEGGINING', () => {
```
Comment (workshur) [inline kafka-ui-react-app/src/components/Topics/Topic/Details/Messages/__test__/utils.spec.ts:77]: returns what?

Comment (workshur) [reply kafka-ui-react-app/src/components/Topics/Topic/Details/Messages/__test__/utils.spec.ts:77]: pls check all the cases

File: kafka-ui-react-app/src/components/Topics/Topic/Details/Messages/__test__/utils.spec.ts [hunk 7]
```diff
@@ -30,4 +39,82 @@ describe('utils', () => {
...
+        { label: '42', value: 42 },
+      ]);
+    });
+    it('returns parsed partition from params when partition NOT includes selected partition', () => {
+      searchParams.set('seekTo', '24::0');
+      expect(getSelectedPartitionsFromSeekToParam(searchParams, part)).toEqual(
+        []
+      );
+    });
+    it('returns partition when searchParams not defined', () => {
```
Comment (workshur) [inline kafka-ui-react-app/src/components/Topics/Topic/Details/Messages/__test__/utils.spec.ts:113]:
```suggestion
    it('returns partitions when searchParams not defined', () => {
```

**Comment by NelyDavtyan** [comment]
Good job,
I'm approving

**Review by NelyDavtyan — APPROVED** [review]

**Review by workshur — APPROVED** [review]
LGTM! Good to go

**Review by Motherships — APPROVED** [review]

**Pull request merged**